src/overlay/OverlayManagerImpl.h

```diff
@@ -227,5 +228,6 @@ class OverlayManagerImpl : public OverlayManager
     void extractPeersFromMap(std::map<NodeID, Peer::pointer> const& peerMap,
                              std::vector<Peer::pointer>& result);
     void shufflePeerList(std::vector<Peer::pointer>& peerList);
+    std::pair<uint32_t, uint32_t> getFlowControlBytesConfig() const override;
```

Comment:
Nit: This std::pair interface is a bit hard to follow. Maybe break this into a struct or typedef? Or at least add a comment on the return type.

Reply:
good idea, using pair of the same type is also kind of error-prone. switched to a struct